generators/generator-base-private.js

```diff
@@ -1139,12 +1139,15 @@ module.exports = class extends Generator {
      * @param {Array|Object} fields - array of fields
      * @param {Array|Object} relationships - array of relationships
      * @param {string} dto - dto
+     * @param {boolean} embedded
      * @returns variablesWithTypes: Array
      */
-    generateEntityClientFields(pkType, fields, relationships, dto, customDateType = 'Moment') {
+    generateEntityClientFields(pkType, fields, relationships, dto, embedded, customDateType = 'Moment') {
```

Comment:
Sorry but I don't understand what you mean by that. I've added the new argument in all usages, so it shouldn't cause any problem. Unless there is some usage I'm not aware of (in blueprint I guess ?).

I don't get your proposal either.

Reply:
It means that adding `embedded` before `customDateType` will make old blueprints use embedded instead of customDateType correct value.
This is a breaking change, embedded should be added after customDateType.

The suggestion is to create an object that will contain every parameter as attribute.

Reply:
Ok, I understand. But creating an object holding parameters is also a breaking change, so it is not for this PR.

I will add `embedded` after `customDateType` with default value to `false`in order to not break actual blueprints!